pkg/runtime/helper.go

```diff
@@ -169,3 +169,13 @@ func (m MultiObjectTyper) IsUnversioned(obj Object) (bool, bool) {
 	}
 	return false, false
 }
+
+// SetZeroValue would set the object of objPtr to zero value of its type.
```

Comment:
Sorry - my bad.
I meant to move it to:
pkg/conversion/helper.go
[Next to EnforcePtr function - which is from the same category].